{"Modified":"2022-12-23T20:34:00","Published":"2022-12-19T21:15:00","access":{},"assigner":"psirt@us.ibm.com","capec":[],"cvss":null,"cwe":"CWE-918","id":"CVE-2022-38708","impact":{},"last-modified":"2022-12-23T20:34:00","references":["https://www.ibm.com/support/pages/node/6841801","https://exchange.xforce.ibmcloud.com/vulnerabilities/234180"],"summary":"IBM Cognos Analytics 11.1.7 11.2.0, and 11.2.1 could be vulnerable to a Server-Side Request Forgery Attack (SSRF) attack by constructing URLs from user-controlled data. This could enable attackers to make arbitrary requests to the internal network or to the local file system. IBM X-Force ID: 234180.","vulnerable_configuration":[{"id":"cpe:2.3:a:ibm:cognos_analytics:11.1.0:*:*:*:*:*:*:*","title":"cpe:2.3:a:ibm:cognos_analytics:11.1.0:*:*:*:*:*:*:*"},{"id":"cpe:2.3:a:ibm:cognos_analytics:11.1.1:*:*:*:*:*:*:*","title":"cpe:2.3:a:ibm:cognos_analytics:11.1.1:*:*:*:*:*:*:*"},{"id":"cpe:2.3:a:ibm:cognos_analytics:11.1.2:*:*:*:*:*:*:*","title":"cpe:2.3:a:ibm:cognos_analytics:11.1.2:*:*:*:*:*:*:*"},{"id":"cpe:2.3:a:ibm:cognos_analytics:11.1.3:*:*:*:*:*:*:*","title":"cpe:2.3:a:ibm:cognos_analytics:11.1.3:*:*:*:*:*:*:*"},{"id":"cpe:2.3:a:ibm:cognos_analytics:11.1.4:*:*:*:*:*:*:*","title":"cpe:2.3:a:ibm:cognos_analytics:11.1.4:*:*:*:*:*:*:*"},{"id":"cpe:2.3:a:ibm:cognos_analytics:11.1.5:*:*:*:*:*:*:*","title":"cpe:2.3:a:ibm:cognos_analytics:11.1.5:*:*:*:*:*:*:*"},{"id":"cpe:2.3:a:ibm:cognos_analytics:11.1.6:*:*:*:*:*:*:*","title":"cpe:2.3:a:ibm:cognos_analytics:11.1.6:*:*:*:*:*:*:*"},{"id":"cpe:2.3:a:ibm:cognos_analytics:11.1.7:*:*:*:*:*:*:*","title":"cpe:2.3:a:ibm:cognos_analytics:11.1.7:*:*:*:*:*:*:*"},{"id":"cpe:2.3:a:ibm:cognos_analytics:11.1.7:-:*:*:*:*:*:*","title":"cpe:2.3:a:ibm:cognos_analytics:11.1.7:-:*:*:*:*:*:*"},{"id":"cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack1:*:*:*:*:*:*","title":"cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack1:*:*:*:*:*:*"},{"id":"cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack2:*:*:*:*:*:*","title":"cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack2:*:*:*:*:*:*"},{"id":"cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack3:*:*:*:*:*:*","title":"cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack3:*:*:*:*:*:*"},{"id":"cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack4:*:*:*:*:*:*","title":"cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack4:*:*:*:*:*:*"},{"id":"cpe:2.3:a:ibm:cognos_analytics:11.2.0:*:*:*:*:*:*:*","title":"cpe:2.3:a:ibm:cognos_analytics:11.2.0:*:*:*:*:*:*:*"},{"id":"cpe:2.3:a:ibm:cognos_analytics:11.2.1:*:*:*:*:*:*:*","title":"cpe:2.3:a:ibm:cognos_analytics:11.2.1:*:*:*:*:*:*:*"},{"id":"cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack5:*:*:*:*:*:*","title":"cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack5:*:*:*:*:*:*"}],"vulnerable_configuration_cpe_2_2":[],"vulnerable_product":["cpe:2.3:a:ibm:cognos_analytics:11.1.0:*:*:*:*:*:*:*","cpe:2.3:a:ibm:cognos_analytics:11.1.1:*:*:*:*:*:*:*","cpe:2.3:a:ibm:cognos_analytics:11.1.2:*:*:*:*:*:*:*","cpe:2.3:a:ibm:cognos_analytics:11.1.3:*:*:*:*:*:*:*","cpe:2.3:a:ibm:cognos_analytics:11.1.4:*:*:*:*:*:*:*","cpe:2.3:a:ibm:cognos_analytics:11.1.5:*:*:*:*:*:*:*","cpe:2.3:a:ibm:cognos_analytics:11.1.6:*:*:*:*:*:*:*","cpe:2.3:a:ibm:cognos_analytics:11.1.7:*:*:*:*:*:*:*","cpe:2.3:a:ibm:cognos_analytics:11.1.7:-:*:*:*:*:*:*","cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack1:*:*:*:*:*:*","cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack2:*:*:*:*:*:*","cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack3:*:*:*:*:*:*","cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack4:*:*:*:*:*:*","cpe:2.3:a:ibm:cognos_analytics:11.2.0:*:*:*:*:*:*:*","cpe:2.3:a:ibm:cognos_analytics:11.2.1:*:*:*:*:*:*:*","cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack5:*:*:*:*:*:*"]}